services/dbaas/cluster.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	mariadb "github.com/ionos-cloud/sdk-go-dbaas-maria"
```

Comment:
We should really start subpackaging these dbaas files on a per-product basis otherwise they will become unmaintainable as more database implementations are added.